docs/source/examples/external_sp2_output.txt

```diff
@@ -1,10 +1,10 @@
 
 Cost
 ----
- 0.7854
+ 0.7071
 
 Free Variables
 --------------
 x : 0.7854
-y : 0.7854
+y : 0.7071
```

Comment:
Wait, shouldn't that number be pi/4? I think this should be `0.7854`, so master's the correct one